### DISCUSSION TITLE
Adding meta data to non-blog pages

### DISCUSSION THREAD

**Discussion by janban-outlook** [squidfunk/mkdocs-material#6754] [Q&A] [answered]
Is there a way to display meta data for normal pages too?
Like in this screenprint of my old Hugo generated web site:

![image](https://github.com/squidfunk/mkdocs-material/assets/43573150/159c8c19-e137-49d1-9842-13b26d4b3bf2)

By the way, the direct link to translations is a nice feature, but not a must-have for me.

**Comment by squidfunk** [comment, marked as answer]
This is not supported and not planned. However, you can achieve this through [customization](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme) – you can basically add arbitrary keys to the front matter of each page and refer to those values in templates.

**Reply by janban-outlook** [reply]
Okay, nice. Customization is an aspect of the product I haven't really explored yet. I'll dive into it. Knowing myself, I need to be careful not to get too enthusiastic and over-customize it 😂.

**Reply by squidfunk** [reply]
Yeah, we try to provide native support for as many common use cases as possible but some things don't generalize really well. That's why we put a huge focus on making and keeping it customizable ☺️ If you're having trouble doing so, don't hesitate to ask for help on our discussion board, as that's what it's for.
